docs/source/quicktour.md

```diff
@@ -0,0 +1,101 @@
+# Quicktour
+
+We provide two main entry points to evaluate models:
+
+- `lighteval accelerate` : evaluate models on CPU or one or more GPUs using [🤗
+  Accelerate](https://github.com/huggingface/accelerate)
+- `lighteval nanotron`: evaluate models in distributed settings using [⚡️
+  Nanotron](https://github.com/huggingface/nanotron)
+
+## Accelerate
+
+### Evaluate a model on a GPU
+
+To evaluate `GPT-2` on the Truthful QA benchmark, run:
+
+```bash
+lighteval accelerate \
+     --model_args "pretrained=gpt2" \
+     --tasks "leaderboard|truthfulqa:mc|0|0" \
+     --override_batch_size 1 \
+     --output_dir="./evals/"
+```
+
+Here, `--tasks` refers to either a comma-separated list of supported tasks from
```

Comment:
maybe also mention task groups https://github.com/huggingface/lighteval/pull/308

Reply:
Once the PR is merged yeah I will add it